### ISSUE TITLE
How to disable subtitle

### ISSUE THREAD

**Issue by AhmadFalahian** [Joran-Dob/flutter_subtitle_wrapper#70]
How to disable subtitle playback?
I did this with the detach() function, but the playback did not stop

`subtitleController.detach();`

**Comment by Joran-Dob** [comment]
Hi, 

I overlooked this project for a bit! Trying to find some time this week to resolve some of the issues or provide some input. Thanks for the information 🚀

**Comment by hardiklakhalani** [comment]
I see the showSubtitle parameter is `final` so it's not letting toggle the value. what could be the purpose to not let it toggle?

**Comment by batoul-alani** [comment]
is there any solution for this issue?

**Comment by braddonako** [comment]
@hardiklakhalani , have you found a way to toggle the subtitles? I am looking toggle the captions as well, but have had no luck.

**Comment by Gameworl** [comment]
hello, any update on this issue ?

**Comment by ArturAntin** [comment]
had the same problem, quick and super hacky solution: make text color and background transparent when you want to hide it.

`SubtitleWrapper(
    videoChild: VideoPlayer(_videoController),
    subtitleController: _subtitleController,
    subtitleStyle: SubtitleStyle(
      textColor: _subtitlesOn
          ? Colors.white
          : Colors.transparent,
      fontSize: 20,
      position: SubtitlePosition(
        bottom: 90,
        right: 16,
        left: 16,
      ),
    ),
    backgroundColor: _subtitlesOn
        ? Colors.black.withOpacity(.5)
        : Colors.transparent,
    videoPlayerController: _videoController,
  ),`

**Comment by Joran-Dob** [comment]
Will close for now as duplicate of #50
